### DISCUSSION TITLE
Disable Coop

### DISCUSSION THREAD

**Discussion by ilker** [BG-Software-LLC/SuperiorSkyblock2#1095] [Pending Feature Requests] [answered]
A setting can be added to turn off coop. 

You said you would do this on Discord.
https://i.hizliresim.com/1440pmb.png

**Comment by OmerBenGera** [comment]
I looked at the code of the plugin again and it won't be as easy as I throught it would be.
Also, island leaders would always be able to give other players the ability to build on their island using the `/is permissions` system.
Therefore, I am not gonna add this atm. However, a PR is more than welcomed :)

**Comment by Lemvs** [comment, marked as answer]
Implemented for a long time.

@OmerBenGera you can mark this answered and close it.
